src/xcode/ENA/ENAUITests/TraceLocations/ENAUITests_10_TraceLocations.swift

```diff
@@ -56,4 +63,42 @@ class ENAUITests_10_TraceLocations: XCTestCase {
 		XCTAssertFalse(app.buttons[AccessibilityIdentifiers.General.primaryFooterButton].exists)
 	}
 
+	func test_screenshot_traceLocation_print_flow() throws {
+		app.launch()
+
+		// navigate to tracelocation card
+		app.swipeUp(velocity: .fast)
+		
```

Comment:
As mentioned before, this is not needed and works fine, without the `swipe`.
```suggestion
```

Reply:
I need this swipe thingy because I am taking screenshot of the card.

Reply:
Just check for existence, wait and it should scroll up. This is guaranteed to show the cell. Swiping might fail if we change the devices.

Reply:
Fixed